site/gatsby-site/i18n/locales/es/translation.json

```diff
@@ -63,25 +63,25 @@
   "Delete": "Borrar",
   "AI Translated": "Traducido por IA",
   "View Original": "Ver Original",
-  "Search over 1600 reports of AI harms": "Busca en más de 1600 reportes de daños de IA",
+  "Search over 1600 reports of AI harms": "Busca en más de 1600 informes de daños de IA",
   "Search 1600+ AI harm reports": "Busca +1600 daños de IA",
-  "Search 1600+ reports": "Buscar +1600 Reportes",
-  "Search reports": "Buscar Reportes",
+  "Search 1600+ reports": "Buscar +1600 Informes",
+  "Search reports": "Buscar Informes",
   "Discover Incidents": "Descubrir Incidentes",
-  "Submit Incident Reports": "Enviar Reportes de Incidentes",
+  "Submit Incident Reports": "Enviar Informes de Incidentes",
   "All Incidents in List Form": "Todos los Incidentes en una Lista",
   "Submission Leaderboard": "Ranking de Reportadores",
   "Classifications View": "Vista de Clasificationes",
-  "Flagged Incidents": "Reportes Denunciados",
+  "Flagged Incidents": "Informes Denunciados",
```

Comment:
Shouldn't this be "Incidentes Denunciados"?

Reply:
yeah no,  the term is used interchangeably there users do flag reports